### PR TITLE
MB-11057 Don't show billable weights warning flags for NTSR

### PR DESCRIPTION
**Pull request by mlm55**
## Summary

TIO shouldn't see “Over weight” or “Missing weight” warnings for NTS-release shipments in the Billable weights section of the Payment Requests page for a move.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. In your terminal, run `DEVSEED_SUBSCENARIO=nts_and_ntsr make db_dev_e2e_populate`
2. Login to the office app as a TOO and look at `HNRMIS`
3. Use the console to get the shipment ids for both shipments and nullify the `prime_estimated_weight` field for both in your DB
4. Use the prime simulator to create a payment request (just add the move management service item)
5. Change user role to TIO and verify that there are no warnIng flags for the NTS-release in the billable weights section

## Verification Steps for Author

These are to be checked by the author.

- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### CODE REVIEW

**Review by kylehilltruss — APPROVED** [review]
Code looks good, functionality works locally.

**Review by jacquelineIO — APPROVED** [review]
Ran locally as expected

File: src/components/ShipmentList/ShipmentList.test.jsx [hunk 1]
```diff
@@ -99,15 +100,17 @@ describe('Shipment List being used for billable weight', () => {
 
   it('does not display weight flags when not appropriate', () => {
     const shipments = [
-      { id: '0001', shipmentType: 'HHG', calculatedBillableWeight: 5666, primeEstimatedWeight: 5600 },
+      { id: '0001', shipmentType: SHIPMENT_OPTIONS.HHG, calculatedBillableWeight: 5666, primeEstimatedWeight: 5600 },
       {
         id: '0002',
-        shipmentType: 'HHG',
+        shipmentType: SHIPMENT_OPTIONS.HHG,
```
Comment (jenniferlynparsons) [inline src/components/ShipmentList/ShipmentList.test.jsx:106]: Thank you for cleaning these all up!

**Review by jenniferlynparsons — APPROVED** [review]
Looks good and works as expected.

**Pull request merged**